docs_user/modules/proc_adopting-autoscaling.adoc

```diff
@@ -77,30 +49,14 @@ oc exec -t $AODH_POD -c aodh-api -- cat /etc/aodh/aodh.conf
 +
 ----
 openstack endpoint list | grep aodh
-| 6a805bd6c9f54658ad2f24e5a0ae0ab6 | regionOne | aodh      | network      | True    | public    | http://aodh-public-openstack.apps-crc.testing  |
-| b943243e596847a9a317c8ce1800fa98 | regionOne | aodh      | network      | True    | internal  | http://aodh-internal.openstack.svc:9696        |
-| f97f2b8f7559476bb7a5eafe3d33cee7 | regionOne | aodh      | network      | True    | admin     | http://192.168.122.99:9696                     |
+| d05d120153cd4f9b8310ac396b572926 | regionOne | aodh  | alarming  | True    | internal  | http://aodh-internal.openstack.svc:8042  |
+| d6daee0183494d7a9a5faee681c79046 | regionOne | aodh  | alarming  | True    | public    | http://aodh-public.openstack.svc:8042    |
 ----
 
-. Create sample resources. You can test whether you can create alarms:
-+
-----
-openstack alarm create \
---name low_alarm \
---type gnocchi_resources_threshold \
---metric cpu \
---resource-id b7ac84e4-b5ca-4f9e-a15c-ece7aaf68987 \
---threshold 35000000000 \
---comparison-operator lt \
---aggregation-method rate:mean \
---granularity 300 \
---evaluation-periods 3 \
---alarm-action 'log:\\' \
---ok-action 'log:\\' \
---resource-type instance
-----
+.Autoscaling template adoption
+
+* `PrometheusAlarm` alarm type must be used instead of `GnocchiAggregationByResourcesAlarm`
```

Comment:
I think we would like to  have an "openstack alarm create" example here in the same way we had in the older documentation.